modules/kernel/src/main/scala/org/tesselation/kernel/Stack.scala

```diff
@@ -0,0 +1,38 @@
+package org.tesselation.kernel
+
+import cats.syntax.all._
```

Comment:
If possible, please import explicitly what's needed.

Reply:
Ok, so we are changing the approach right? So far we were importing everything.